Origin: Gentoo
Description: Follow up to 80_fix-pickle.patch.  Just disable pickling
 altogether since the previous patch doesn't really fix the problem, and not
 storing the pickle seems to have no adverse effects on the build, while
 avoiding the observed traceback.
Author: Barry Warsaw <barry@debian.org>
Forwarded: no

--- a/waflib/Build.py
+++ b/waflib/Build.py
@@ -151,6 +151,7 @@
				f.close()
		self.init_dirs()
	def store(self):
+		return
		data={}
		for x in SAVED_ATTRS:
			data[x]=getattr(self,x)
